schemas/top-api.yml

```diff
@@ -438,6 +438,155 @@ paths:
           $ref: '#/components/responses/UnexpectedError'
       tags:
         - conceptcluster
+  /conceptgraphs/pipeline:
+    parameters:
+      - $ref: '#/components/parameters/include'
+    post:
+      operationId: startConceptGraphPipeline
+      summary: 'Initializes a complete pipeline with all steps ''preprocessing'', ''embedding'', ''clustering'' & ''graph creation'''
+      parameters:
+        - name: process
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: lang
+          in: query
+          schema:
+            type: string
+        - name: skipPresent
+          in: query
+          schema:
+            type: boolean
+            default: true
+        - name: returnStatistics
+          in: query
+          schema:
+            type: boolean
+            default: false
+      requestBody:
+        description: Upload configs and data file.
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                data:
+                  type: string
+                  format: binary
+                labels:
+                  type: string
+                  format: binary
+                dataConfig:
+                  type: string
+                  format: binary
+                embeddingConfig:
+                  type: string
+                  format: binary
+                clusteringConfig:
+                  type: string
+                  format: binary
+                graphConfig:
+                  type: string
+                  format: binary
+              required:
+                - data
+      responses:
+        '200':
+          description: A short summary of the graphs created by this pipeline
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PipelineResponse'
+        '202':
+          description: The POST request was accepted and a pipeline process started
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PipelineResponse'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+      tags:
+        - conceptgraphs
+  /conceptgraphs/statistics:
+    parameters:
+      - $ref: '#/components/parameters/include'
+    get:
+      operationId: getConceptGraphStatistics
+      summary: Gets an overview of the concept graphs for a process
+      parameters:
+        - name: process
```

Comment:
What happens if this parameter is missing? Is it required and should be converted to a path parameter?

Reply:
Also: there might be no process for a given ID. This should return 404.

Reply:
if this parameter is missing, it tries to fetch a process named  'default' (that's also true for creation etc.). If no process is available it returns json with an error message. But there should be a 404 response as well...

Reply:
I've added 404 for this specific case in [c310534](https://github.com/Onto-Med/top-api/pull/163/commits/c310534410bb952b7363f5bb7c96c8f81adbd5d7)

Reply:
Parameter `process` now has a default value 'default'.